bench/micro/Bench/Database/LSMTree/Normal.hs

```diff
@@ -38,13 +38,16 @@ benchmarks = bgroup "Bench.Database.LSMTree.Normal" [
   Types
 -------------------------------------------------------------------------------}
 
+deriving via ResolveAsFirst Word64 instance ResolveValue Word64
```

Comment:
Do we need this instance?

Reply:
Unfortunately, that instantiation is an orphan, which causes problems when compiling with `-Werror`. I’ve changed `benchInsertBatches` to use `K` and `V2` instead of `Word64`, which not only removes that problem but also makes the code nicer. The respective changes are in be74f4c00b3042ea28d9f51791bbfdbe1c72f90d.